src/main/java/org/terasology/gooeyDefence/health/HealthSystem.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2018 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.gooeyDefence.health;
+
+import org.terasology.entitySystem.entity.EntityRef;
+import org.terasology.entitySystem.event.ReceiveEvent;
+import org.terasology.entitySystem.systems.BaseComponentSystem;
+import org.terasology.entitySystem.systems.RegisterSystem;
+import org.terasology.gooeyDefence.events.health.DamageEntityEvent;
+import org.terasology.gooeyDefence.events.health.EntityDeathEvent;
+
+/**
+ * Handles operations involving health on entities.
+ *
+ * @see HealthComponent
+ */
+@RegisterSystem
+public class HealthSystem extends BaseComponentSystem {
```

Comment:
May also be ideal to rename this. IMHO either some module should enhance an existing _generic_ system of the same nature if something is missing that's comparatively generic *or* develop a more specific _flavorful_ variant (like the goo system I mention)

Helps fight fragmentation, prevent confusion, as well as adding more uniqueness when you do make something similar but distinct.

Reply:
I may rename it to BasicHealth or something that better reflects the simplistic nature of it.
Alternativly I'll pull health out of core and in doing so remove my version.